qemu/tests/flow_caches_stress_test.py

```diff
@@ -153,13 +154,17 @@ def get_if_queues(ifname):
                               % netperf_timeout, logging.info)
         test_option = "-t TCP_CRR -l %s -- -b 10 -D" % netperf_timeout
         netperf_client.bg_start(host_ip, test_option, client_num)
-
-        utils_misc.wait_for(lambda: not netperf_client.is_netperf_running(),
-                            timeout=netperf_timeout, first=590, step=2)
+        if utils_misc.wait_for(lambda: not netperf_client.is_netperf_running(),
```

Comment:
My original intention： If netperf stops running before it timeout. python can get the test results for the case normally. Rather than as before, the case may have stopped midway, but python did not catch it, and mistakenly thought it was a test pass.
From this point of view, I think this is reasonable. Please correct me if I'm wrong.

Reply:
Hello @PaulYuuu  I rewrote the judgment method of netperf_timeout, could you please help me review it again? Thanks in advance.